src/components/ListItem.jsx

```diff
@@ -120,26 +119,46 @@ export function ListItem({
 	};
 
 	return (
-		<li className={isChecked ? checkedItemStyle : uncheckedItemStyle}>
-			<input
-				type="checkbox"
-				id={`checkbox-${id}`} // Unique identifier
-				name={name}
-				onChange={handleChecked}
-				checked={isChecked}
-			></input>
-			<label className="flex px-2 items-center w-full h-full text-lg xsm:text-xs sm:text-sm">
-				{capitalizeFirstLetterOfEachWord(name)}
-			</label>
-			<span
-				className={`px-2 py-1 mx-2 rounded-lg min-w-fit`}
-				style={{ backgroundColor: tagColor }}
+		<li>
+			<div
+				htmlFor={`checkbox-${id}`}
+				onClick={handleChecked}
+				className={`cursor-pointer bg-white shadow hover:shadow-md h-[72px] flex items-center justify-between rounded-lg p-6 transition-shadow duration-300 ease-in-out`}
+				onKeyPress={handleKeyPress}
```

Comment:
Oh, also, onKeyPress is showing up deprecated for me.

Reply:
Weird, that was what eslint a11y suggested https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/0d5321a5457c5f0da0ca216053cc5b4f571b53ae/docs/rules/click-events-have-key-events.md 
I need to use it to make the whole list item useable with the tab button. I was having issues wrapping everything with the label. Let me know if you can think of a better fix!